src/components/Card/Activity.js

```diff
@@ -34,3 +35,13 @@ const NameStyled = styled.li`
 const SpanStyled = styled.span`
   padding-left: 10px;
 `
+
+const BtnDelete = styled.span`
```

Comment:
I think it is better to not call it Btn in case of semantic. Maybee something like IconDelete makes more sense.